### ISSUE TITLE
How to get inference_config parameters while finetune on my own dataset?

### ISSUE THREAD

**Issue by wsxwd** [KwaiVGI/VideoAlign#3]

**Comment by GongyeLiu** [comment]
Hi, @wsxwd 

The inference_config is primarily used to normalize the reward values. This configuration is optional, the inference code should works fine without it. 

https://github.com/KwaiVGI/VideoAlign/blob/f9a9654bad88a720e82c4aacc57bc566ce404eb5/inference.py#L67-L74


In practice, we perform inference on all the samples in the validation set to compute the mean and standard deviation in inference_config. You might consider applying a similar strategy for your dataset.

I hope this helps!
